src/vs/workbench/browser/parts/editor/editor.ts

```diff
@@ -41,6 +42,24 @@ export const DEFAULT_EDITOR_PART_OPTIONS: IEditorPartOptions = {
 	splitSizing: 'distribute'
 };
 
+export function computeEditorAriaLabel(input: IEditorInput, group: IEditorGroup | undefined, groupCount: number): string {
+	let ariaLabel = input.ariaLabel;
+	if (group && !group.isPinned(input)) {
+		ariaLabel += localize('preview', ", preview");
```

Comment:
I have a feeling translators will not know how to translate a leading ", ", I would remove it from the NLS

Reply:
Thanks, tackled now.